src/vm/method.hpp

```diff
@@ -1396,7 +1396,7 @@ class MethodDesc
     void RecordAndBackpatchEntryPointSlot_Locked(LoaderAllocator *mdLoaderAllocator, LoaderAllocator *slotLoaderAllocator, TADDR slot, EntryPointSlots::SlotType slotType, PCODE currentEntryPoint);
 
 public:
-    void MethodDesc::BackpatchEntryPointSlots(PCODE entryPoint)
+    void BackpatchEntryPointSlots(PCODE entryPoint)
```

Comment:
this is fixed on mainline.

Reply:
You mean #22810? That was not merged to master. I have revert this file.